src/dowel/logger.py

```diff
@@ -213,6 +214,48 @@ def log(self, data):
                     type(data).__name__))
             self._warn(warning)
 
+    def reorganize_file(self,data,file_name):
```

Comment:
I think it would be better to handle this automatically when the keys change in a `CsvOutput`.